contracts/bridge/x/v3/CoreV3.sol

```diff
@@ -2,7 +2,7 @@
 // solhint-disable compiler-version
 pragma solidity ^0.8.11;
 
-interface Core {
+interface CoreV3 {
```

Comment:
Add a copyright notice as the first line of this file:
// Copyright (c) Immutable Australia Pty Ltd 2018 - 2024

Reply:
@drinkcoffee should I remove the MIT license?

Reply:
I've seen these annotations in a few files:
```solidity
// Copyright (c) Immutable Pty Ltd 2018 - 2024
// SPDX-License-Identifier: Apache-2
```